host/frontend/liboperator/operator/operator.go

```diff
@@ -97,9 +99,45 @@ func CreateHttpHandlers(
 	router.HandleFunc("/infra_config", func(w http.ResponseWriter, r *http.Request) {
 		ReplyJSONOK(w, config)
 	}).Methods("GET")
+	remoteUrl, proxy, err := getOpenWrtProxy()
+	if err == nil {
+		router.HandleFunc("/openwrt", forwardWithoutUrl(remoteUrl, proxy))
+		// We cannot set up the base url for luci, so just add necessary paths as well.
+		router.HandleFunc("/cgi-bin/{rest:.*}", forwardWithPreservedUrl(remoteUrl, proxy))
+		router.HandleFunc("/luci-static/{rest:.*}", forwardWithPreservedUrl(remoteUrl, proxy))
```

Comment:
Looks like you're not using the `rest` variable anywhere so the matcher for this route could simply be `"/luci-static/"` I think.
The same applies for cgi-bin.

Reply:
yeah, thanks for catching that, I replaced it with PathPrefix